<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pre2</class>
 <widget class="QDialog" name="Pre2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>313</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>579</width>
    <height>313</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>579</width>
    <height>313</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Show Step</string>
  </property>
  <widget class="QLabel" name="label_exp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Expression</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGo">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>275</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Go</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="stackTable2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>491</width>
     <height>71</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>Stack 3</string>
    </property>
   </row>
  </widget>
  <widget class="QLineEdit" name="lneExp">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTableWidget" name="stackTable1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>61</y>
     <width>491</width>
     <height>110</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>Stack 1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Stack 2</string>
    </property>
   </row>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
